File: Modul 6/C/PRAK602-2310817310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h>

void main() {
    int TOTAL;

    scanf("%d", &TOTAL);
    
    int arr[TOTAL];

    for(int i = 0 ; i < TOTAL ; i++) {
        scanf("%d", &arr[i]);
    }
    
    for(int i = 0 ; i < TOTAL ; i++) {
        arr[i] = arr[i] * (1 + i);
        printf("%d ", arr[i]);
    }
}

File: Modul 6/Python/PRAK602-2310817310005-MuhammadNurwahyudiAdhitama.py
import numpy as np

TOTAL = int(input(""))

E_ARRAY = list(map(int,input("").split()))

arr = np.array(E_ARRAY).reshape((1,TOTAL))

i = 0

while i < TOTAL :
    HASIL = E_ARRAY[i] * (i + 1)
    print(f"{HASIL}", end = " ")
    i += 1